src/main/resources/templates/subscription.html

```diff
@@ -156,7 +156,22 @@ <h3 class="modal-title text-center" id="formHeader">Subscription Form</h3>
                                     </label>
                                 </div>
                             </div>
-
+                            
+                             <!-- Form post emailSubject field -->                            
+                            <div class="form-group" data-bind="visible: $data.notificationType() == 'MAIL'">
```

Comment:
if you merge with current master you will see that a H5 class is added for the labels.

Reply:
done